rholang/src/main/scala/coop/rchain/rholang/interpreter/Substitute.scala

```diff
@@ -77,7 +78,7 @@ object Substitute {
       env.get(term.index) match {
         case Some(par) => Applicative[M].pure(Right(par))
         case None =>
-          interpreterErrorM[M].raiseError(SubstituteError(s"Illegal VarRef [$term]"))
+          Applicative[M].pure(Left(term))
```

Comment:
Good catch. At the time it seemed to make perfect sense that they would all be available right now, but of course some of them could not be available yet.

Reply:
I should have caught that in the review.